Gemfile.lock

```diff
@@ -1242,7 +1242,7 @@ GEM
       multipart-post
       oauth2
     ruby-progressbar (1.13.0)
-    ruby-saml (1.16.0)
+    ruby-saml (1.17.0)
```

Comment:
Gemfile.lock shouldn't exist in the knapsack, according to Rob

Reply:
alright sounds good i can delete the whole thing then